src/benchmarks/micro/libraries/System.Runtime.Extensions/Perf.Convert.cs

```diff
@@ -77,6 +77,18 @@ public DateTime ToDateTime_String(string value)
         [Benchmark]
         public byte[] FromBase64Chars() => Convert.FromBase64CharArray(_base64Chars, 0, _base64Chars.Length);
 
+#if NET5_0_OR_GREATER
+        [GlobalSetup(Target = nameof(ToHexString))]
+        public void SetupToHexString()
+        {
+            _binaryData = new byte[64];
+            new Random(42).NextBytes(_binaryData);
```

Comment:
we have a helper type for that:

```suggestion
            _binaryData = ValuesGenerator.Array<byte>(64);
```

you will most probably need to add `using BenchmarkDotNet.Extensions;` at the top of the file